### PR TITLE
Fix the example cheatcode

### PR DESCRIPTION
**Pull request by piotmag769**
We had not been waiting for async test function to execute.

### CODE REVIEW

File: tests/integration/fuzzing/parameterized_test.cairo [hunk 1]
```diff
@@ -82,3 +82,21 @@ func test_check_exact_example{syscall_ptr: felt*, range_check_ptr}(a: felt, b: f
...
+        example(a=11)
+    %}
+    return ();
+}
+
+@external
+func test_negative_example{syscall_ptr: felt*, range_check_ptr}(a: felt) {
+    %{
+        assert ids.a == 12
+    %}
```
Comment (piotmag769) [inline tests/integration/fuzzing/parameterized_test.cairo:98]:
```suggestion
    assert a = 12;
```

File: protostar/testing/environments/fuzz_test_execution_environment.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import asyncio
 import dataclasses
 from asyncio import to_thread
```
Comment (piotmag769) [inline protostar/testing/environments/fuzz_test_execution_environment.py:3]: Maybe we could use asyncio.to_thread instead since we already import asyncio

**Review by piotmag769 — COMMENTED** [review]
Some nitpicks

File: tests/integration/fuzzing/parameterized_test.cairo (hunk 1 above)
Comment (Arcticae) [reply tests/integration/fuzzing/parameterized_test.cairo:98]: Actually that should do the job. Good idea.

**Review by mkaput — APPROVED** [review]
Yep, this looks to be the smallest fix possible for this

**Pull request merged**